systemvm/debian/opt/cloud/bin/cs/CsAddress.py

```diff
@@ -477,8 +477,8 @@ def fw_vpcrouter(self):
                             ("eth1", "eth1", guestNetworkCidr)])
 
             self.fw.append(["nat", "front",
-                            "-A POSTROUTING -s %s -o %s -j SNAT --to-source %s" %
-                            (guestNetworkCidr, self.dev, self.address['public_ip'])])
+                            "-A POSTROUTING -o %s -j SNAT --to-source %s" %
+                            (self.dev, self.address['public_ip'])])
```

Comment:
@Slair1 to minimize impact/regression from this, can you consider @ustcweizhou 's advise on doing something like : 
```
if self.address["source_nat"]:
    self.fw.append(new rule)
else:
    self.fw.append(old rule)
```

Reply:
I think it might be safer to use @rhtyd suggestion. What do you guys think about it?

Reply:
I agree with you guys. @Slair1 can you please address @rhtyd comment?